src/main/java/mate/academy/library/repository/book/BookSpecificationBuilder.java

```diff
@@ -11,20 +11,22 @@
 @Component
 @RequiredArgsConstructor
 public class BookSpecificationBuilder implements SpecificationBuilder<Book> {
+    private static final String TITLE_SPECIFICATION = "title";
+    private static final String AUTHOR_SPECIFICATION = "author";

```

Comment:
It was decided to use constant field values
https://mate.academy/chats/21985?selectedMessageId=1800943&threadId=1800143